### PR TITLE
Ncnews 2: adding 200 and 404 paths for /api/topcs

### PR DESCRIPTION
**Pull request by garetheverson**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        res.body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  describe('GET /api/[unrecognised path]', () => {
+    test('404: responds with "Route note found" for an unrecognised GET path', () => {
```
Comment (dentednerd) [inline __tests__/app.test.js:27]:
```suggestion
    test('404: responds with "Route not found" for an unrecognised GET path', () => {
```
Just a wee typo 😄

File: app.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const format = require('pg-format');
```
Comment (dentednerd) [inline app.js:2]: You won't need to use pg-format in here!

File: models/topics.models.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const { query } = require('../db/connection');
+const db = require('../db/connection');
```
[the remark this answers is not in the document]
Comment (garetheverson) [reply models/topics.models.js:2]: Thanks @dentednerd removing line 1. Tests still pass

File: __tests__/app.test.js (hunk 1 above)
Comment (garetheverson) [reply __tests__/app.test.js]: Thanks @dentednerd

**Pull request merged**